static/js/chart/draw_choropleth.ts

```diff
@@ -98,23 +104,46 @@ function drawChoropleth(
     .append("svg")
     .attr("viewBox", `0 0 ${chartWidth} ${chartHeight}`)
     .attr("preserveAspectRatio", "xMidYMid meet");
-  const legendWidth = generateLegend(
-    svg,
-    chartWidth,
-    chartHeight,
-    colorScale,
-    unit
-  );
   const map = svg.append("g").attr("class", "map");
 
   // Combine path elements from D3 content.
   const mapContent = map.selectAll("path").data(geoJson.features);
 
   const projection = geo.geoAlbersUsaTerritories();
   const geomap = d3.geoPath().projection(projection);
+  const legendWidth = generateLegend(
+    svg,
+    chartWidth,
+    chartHeight,
+    colorScale,
+    unit
+  );
 
   // Scale and center the map
-  fitSize(chartWidth - legendWidth, chartHeight, geoJson, projection, geomap);
+  if (zoomDcid) {
+    const geoJsonFeature = geoJson.features.find(
+      (feature) => feature.properties.geoDcid === zoomDcid
+    );
+    if (geoJsonFeature) {
+      fitSize(
+        chartWidth - legendWidth,
+        chartHeight,
+        geoJsonFeature,
+        projection,
+        geomap,
+        ZOOMED_SCALE_AMOUNT
+      );
+    }
```

Comment:
how should we handle the case where a feature wasn't found?

Reply:
I guess when a feature isn't found, we can just fit it like we were drawing the map without zooming in?

Reply:
Or maybe we could have an error message that says "Sorry, unable to draw the map for this place"

Reply:
or we can have an error message that says something like "Sorry, unable to draw map for this place"

Reply:
done